frontend/src/components/SharedUsersList/SharedUsersList.tsx

```diff
@@ -18,6 +18,11 @@ export const SharedUsersList: React.FC<Props> = ({ device, connected = [], users
   const { access } = useSelector((state: ApplicationState) => state.accounts)
   const filtered = sort(users.filter(user => !connected.find(_u => _u.email === user.email)))
   const listUserLinked = sort(access.filter(user => !connected.find(_u => _u.email === user.email)))
+  const { devices } = useDispatch<Dispatch>()
+
+  useEffect(() => {
```

Comment:
Please check to make sure we need this. Would be better to to have a fetch here - we should have the data.